README.md

```diff
@@ -5,4 +5,4 @@
 
 # Alex is awesome
 
-# Johannes Nicolai is awesome too
+# Johannes is awesome too
```

Comment:
Great! As everything added dilutes the value of anything else, this is the right approach here!